src/lib/storage/fixed_size_attribute_vector.cpp

```diff
@@ -0,0 +1,49 @@
+// the linter wants this to be above everything else
+#include "fixed_size_attribute_vector.hpp"
+#include <boost/numeric/conversion/cast.hpp>
+#include <types.hpp>
+#include <vector>
+
+namespace opossum {
+
+template <typename T>
+FixedSizeAttributeVector<T>::FixedSizeAttributeVector(const size_t size) {
+  _values = std::vector<T>(size);
+}
+
+// returns the value id at a given position
+template <typename T>
+ValueID FixedSizeAttributeVector<T>::get(const size_t i) const {
+  return ValueID{_values.at(i)};
```

Comment:
This is the main access method for one of the main data structures. As such, it is one of the few places where we do not want bounds checking, at least not in release mode

Reply:
Resolved in e657a507915d73d9da384eb3b4757ee255cd1288